src/frontend/src/handler/create_source.rs

```diff
@@ -51,6 +51,7 @@ pub(crate) fn make_prost_source(
         database_id,
         name,
         info: Some(source_info),
+        owner: session.user_name().to_string(),
```

Comment:
Plz fill the owner field in create_table/create_mv using `session.user_name().to_string()` as also.